src/index.html

```diff
@@ -6,11 +6,57 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Search bar airbnb</title>
+
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      class="form-wrapper first-wrapper"
+      data-qa="big"
+    >
+      <label for="searchOne">
+        <img
+          alt="undefinied"
```

Comment:
Alt attribute should describe the image
```suggestion
          alt="search icon"
```